tests/Unit/Domain/CoordinateMaps/Test_CubicScale.cpp

```diff
@@ -72,18 +71,17 @@ void test_map() {
     double t = -0.5;
     const double dt = 0.6;
 
-    domain::FunctionsOfTime::PiecewisePolynomial<deriv_order> expansion_a(
-        t, init_func_a);
-    domain::FunctionsOfTime::FunctionOfTime& expansion_a_base = expansion_a;
+    using Polynomial =
+        domain::FunctionsOfTime::PiecewisePolynomial<deriv_order>;
+    using FoftPtr = std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>;
```

Comment:
If you're going to make this type alias, why not put it in a wider scope so that it doesn't need to be redeclared multiple times? Also, if it's around, you may as well put it in the `unordered_map` declarations.
Not sure if it's readable enough to be worth bringing all the way up to the src, but it would shorten declarations there substantially.

Reply:
I personally like scoping things, in this case inside the function. I do agree that it should be used in the `unordered_map`s :)